connectors/src/connectors/zendesk/index.ts

```diff
@@ -469,11 +543,33 @@ export class ZendeskConnectorManager extends BaseConnectorManager<null> {
   }
 
   async pause(): Promise<Result<undefined, Error>> {
-    throw new Error("Method not implemented.");
+    const { connectorId } = this;
+    const connector = await ConnectorResource.fetchById(connectorId);
+    if (!connector) {
+      logger.error({ connectorId }, "[Zendesk] Connector not found.");
+      return new Err(new Error("Connector not found"));
+    }
+    await connector.markAsPaused();
+    const result = await this.stop();
+
+    return result.isErr() ? result : new Ok(undefined);
   }
 
   async unpause(): Promise<Result<undefined, Error>> {
-    throw new Error("Method not implemented.");
+    const { connectorId } = this;
+    const connector = await ConnectorResource.fetchById(connectorId);
+    if (!connector) {
+      logger.error({ connectorId }, "[Zendesk] Connector not found.");
+      return new Err(new Error("Connector not found"));
+    }
+    await connector.markAsUnpaused();
+
+    const brandIds = await ZendeskBrandResource.fetchAllBrandIds({
+      connectorId,
+    });
+    const result = await launchZendeskSyncWorkflow({ connector, brandIds });
+
+    return result.isErr() ? result : new Ok(undefined);
```

Comment:
Same here?

Reply:
Here types don't match since we pass the workflow ID in the `Ok` of `launchZendeskSyncWorkflow`, turns out we don't need this workflow ID so i'll just change `launchZendeskSyncWorkflow` and return result everywhere appliable

Reply:
@flvndvd done